solutions/0900-0999/0948-bag-of-tokens-medium.md

```diff
@@ -203,4 +203,54 @@ class Solution {
         return mx_score;
     }
 }
-```
\ No newline at end of file
+```
+
+**Go Lang**
+
+<SolutionAuthor name="@wingkwong"/>
+
+```
+// Idea
+// ------------------
+// Sorting:
+// Why sorting? 
+// because are need max power when we consume score
+// Because least power or most power gives us same score of 1
+// But 1 score can max power, so we have to consumer the power in order of descending from score consumed.
```

Comment:
But 1 score can max power -> But 1 score can maximize power
so we have to consumer the power -> so we have to consume the power